Pflichtenheft.md

```diff
@@ -81,34 +106,50 @@ Anfrage des Kunden per Webseite ist optional
  - Preis
  - Status (geplant, beauftragt, in Durchführung, beendet, abgebrochen)
  - Anmerkungen (z.B. Abbruchgrund)
-- Produktleistungen
-- 
+- Mahnwesen
+ - Auftrag (Link zum Auftrag)
```

Comment:
Das wird ganz interessant weil wir uns hier wirklich überlegen müssen ob es einen FatClient geben soll oder ob wir generell alles über Weboberflächen machen. Ich hab noch keine Webseiten mit C# gebaut, aber ich hab schon vielfach gelesen das das eigentlich ohne Probleme geht wenn man die Darstellung in XAML macht